[TM] Chp3-1 More inputs to the vector attribute data section (#3299)
qgis/QGIS-Documentation: source/docs/training_manual/answers/answers.rst

|RF| *Working with Vector Data*
-------------------------------------------------------------------------------

.. _vector-explore-attribute-data:

|basic| *Attribute data*
...............................................................................

* There should be 9 fields in the :guilabel:`rivers` layer:

  #. Select the layer in the :guilabel:`Layers` panel.

>   #. Right-click and choose :guilabel:`Attribute table`, or press the |openTable|

Reviewer: ```suggestion
  #. Right-click and choose :guilabel:`Open Attribute Table`, or press the |openTable|
```

     button on the :guilabel:`Attributes Toolbar`.
  #. Count the number of columns.

>   .. tip:: A quicker approach could also consist in double-clicking the

Reviewer: ```suggestion
  .. tip:: A quicker approach could be to double-click the
```

>    :guilabel:`rivers` layer, open :menuselection:`Layer properties --> Source

Reviewer: ```suggestion
   :guilabel:`rivers` layer, open the :menuselection:`Layer properties --> Source
```

>    Fields` tab and pick there the count of fields.

Reviewer: ```suggestion
   Fields` tab and count of fields.
```

Author: Missing word in your suggestion @havatv ? 
Your sentence would be  ```.. tip:: A quicker approach could be to double-click the :guilabel:`rivers` layer, open the :menuselection:`Layer properties --> Source Fields` tab and count of fields.```
The idea I had is to simply take note of the max of rows as they are numbered in the table.

Reviewer: If we are talking about rows, we can't use "fields".

Reviewer 2: But "rows" suggests records/features. How about "where you wil find a numbered list of [the table's] fields"?

Reviewer: My mistake - I did not check the context. I have updated.

> * Information about towns are available in the :guilabel:`places` layer. Open its

Reviewer 2: ```suggestion
* Information about towns is available in the :guilabel:`places` layer. Open its
```

>   attribute table as previously seen: there are two features whose :guilabel:`place`

Reviewer: ```suggestion
  attribute table (in the same way as for the:guilabel:`rivers` layer): there are two features whose :guilabel:`place`
```

Reviewer 2: ```suggestion
  attribute table as you did with the :guilabel:`rivers` layer: there are two features whose :guilabel:`place`
```

Reviewer: Fine.

  attribute is set to ``town``: *Swellendam* and *Buffeljagsrivier*.

>   You can add any more information picked from these two records.

Reviewer: I don't understand this sentence.  Should be removed or reformulated.

Author: This is just to skip me commenting each field (eg, IDs are xxx and zzz... with no great interest) of the table for those two features. But given that the instruction is "Tell us about towns" (with no limit), they are giving the opportunity to add any information they could take on those two rows.
Any suggestion of rewording?

>   You can add any more information picked from these two records.

Reviewer: ```suggestion
  You can use other fields from these two records, if you like.
```
